src/indexation/pht.cpp

```diff
@@ -27,21 +27,26 @@ void Pht::lookupStep(Prefix p, std::shared_ptr<int> lo, std::shared_ptr<int> hi,
         }
         else if (is_leaf or *lo > *hi) {
             // leaf node
+            auto to_insert = p.getPrefix(mid);
+            cache_.insert(to_insert);
+            
```

Comment:
Beware empty trailing white spaces. See [this][] or [my vimrc](https://github.com/sim590/dotfiles/blob/master/vimrc#L137).

[this]: http://vim.wikia.com/wiki/Remove_unwanted_spaces